VideoAPI/Video.API/Controllers/ConferenceManagementController.cs

```diff
@@ -97,5 +98,45 @@ public async Task<IActionResult> EndVideoHearingAsync(Guid conferenceId)
                 return StatusCode(ex.StatusCode, ex.Response);
             }
         }
+        
+        /// <summary>
+        /// Transfer a participant in or out of a hearing
+        /// </summary>
+        /// <param name="conferenceId">Id for conference</param>
+        /// <param name="transferRequest">Participant and direction of transfer</param>
+        /// <returns></returns>
+        [HttpPost("{conferenceId}/transfer")]
+        [SwaggerOperation(OperationId = "TransferParticipant")]
+        [ProducesResponseType((int) HttpStatusCode.OK)]
+        public async Task<IActionResult> TransferParticipantAsync(Guid conferenceId, TransferParticipantRequest transferRequest)
+        {
+            _logger.LogDebug("TransferParticipant");
+            try
+            {
+                switch (transferRequest.TransferType)
+                {
+                    case TransferType.Call:
+                        _logger.LogDebug($"Transferring participant {transferRequest.ParticipantId} into hearing room");
```

Comment:
log messages could have custom dimensions for the conference id / participant id

Reply:
i've re-written the logs to take advantage of custom dimensions now. thank you